src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -92,7 +109,33 @@ class VeteranBenefitSummaryLetter extends React.Component {
           recent periods of service.
         </p>
         <h2>Military Service Information</h2>
-        {militaryServiceInfo}
+        <div className="form-checkbox">
+          <input
+              autoComplete="false"
+              id="serviceInfoCheckboxId"
```

Comment:
Yeah I'm not totally positive on what the request body should be, but I think for military service it's looking for this: `"militaryService": false,` (found in this documentation at the bottom: https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/EVSS%20Integration/VA%20Letters/letters-responses-from-vets-api.md). But I see that the other options do have slightly different keys. That'll be great to map those. Thanks!